tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+---
+layout: tutorial
+title: Write Your First Flutter App
+permalink: /tutorials/get-started/
+---
+
+This is a guide to creating your first Flutter app. If you know
+how to program, you can complete this tutorial. You don’t need
+previous experience with Dart or mobile programming.
+
+* TOC
+{:toc}
+
+### What are Flutter and Dart?
+
+_Flutter_ is a mobile app SDK to help developers and designers
```

Comment:
This is now:

Flutter is Google’s mobile UI framework for crafting high-quality native interfaces on iOS and Android in record time. Flutter works with existing code, is used by organizations around the world, and is free and open source.

Reply:
I wonder if we can skip this intro section entirely, and get into the tutorial quicker. I would assume the developer is coming to this from the front page, or in some other way has heard what Flutter is.

Reply:
Done